front-end/gitaly/server/server.go

```diff
@@ -70,17 +72,9 @@ func (s *grpcServer) Start(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-
-	go func() {
-		if err := grpcServer.Serve(l); err != nil {
-			log.Printf("Front-end failed: %s", err)
-		}
-	}()
-
-	go func() {
-		<-ctx.Done()
-		grpcServer.GracefulStop()
-	}()
+	if err := grpcServer.Serve(l); err != nil {
```

Comment:
@stepanvalyavskiy please add a `@todo` [puzzle](https://github.com/yegor256/pdd#how-to-format) to move `Serve` and `GracefulStop` to goroutine